src/distPlus/distribution/Continuous.re

```diff
@@ -36,59 +51,89 @@ let combinePointwise =
   // can just sum them up. Otherwise, all bets are off.
   let combinedIntegralSum =
     Common.combineIntegralSums(
-      knownIntegralSumsFn,
-      t1.knownIntegralSum,
-      t2.knownIntegralSum,
+      integralSumCachesFn,
+      t1.integralSumCache,
+      t2.integralSumCache,
     );
 
+  // TODO: does it ever make sense to pointwise combine the integrals here?
+  // It could be done for pointwise additions, but is that ever needed?
```

Comment:
I could imagine it doing so, but this seems like an optimization to me (and thus not particularly necessary right now). I could imagine bugs in errors we might make, so it's safer to focus a bit more on the "just make sure we can calculate it or at least approximate it very quickly, then always do that" strategy.